### PR TITLE
WRN-3835: Migrate Item, LabeledItem and SelectableItem unit tests to Testing Library

### PR DESCRIPTION
**Pull request by stanca-pop-lgp**
Enact-DCO-1.0-Signed-off-by: Stanca Pop stanca.pop@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Migrated unit tests from Enzyme to Testing Library for Item, LabeledItem and SelectableItem components.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRN-3835

### CODE REVIEW

**Review by daniel-stoian-lgp — APPROVED** [review]
LGTM

File: Item/tests/Item-specs.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+	test('should support adding text as a child', () => {
+		const expected = 'Hello Item';
+		render(<ItemBase>{expected}</ItemBase>);
+		const actual = screen.getByText('Hello Item');
+
+		expect(actual).toHaveTextContent(expected);
+	});
+
+	test('should apply `inline` class when inline', function () {
+		render(<ItemBase inline>Hello Item</ItemBase>);
```
Comment (alexandrumorariu) [inline Item/tests/Item-specs.js:17]: Please replace function () with an arrow functions here and below.

File: SelectableItem/tests/SelectableItem-specs.js [hunk 2]
```diff
@@ -1,32 +1,25 @@
...
+		render(<SelectableItem selected>Hello SelectableItem</SelectableItem>);
 
-		const expected = 1;
-		const actual = selectableItem.find(`.${css.selected}`).length;
+		const expected = 'selected';
+		const actual = screen.getByRole('checkbox').children[0].children[0];
 
-		expect(actual).toBe(expected);
+		expect(actual).toHaveClass(expected);
 	});
```
Comment (alexandrumorariu) [inline SelectableItem/tests/SelectableItem-specs.js]: Please remove this empty line.

**Review by alexandrumorariu — APPROVED** [review]
LGTM

**Pull request merged**